features/user_can_delete_message.feature

```diff
@@ -0,0 +1,35 @@
+Feature: User can empty inbox
+  As a user
+  In order to clean up my inbox
+  I want to be able to delete messages
+
+  Background:
+    Given the following user exist:
+
+      | name   | email          | password  | id |
+      | dmybob | dmybob@bob.bob | bobbobbob | 1  |
+      | dmypip | dmypip@bob.bob | pippippip | 2  |
+
+    Given the inbox has content
+    And I am on the Login page
```

Comment:
I would like you guys to login with the warden helper that is being used in the `user_can_compose_message` feature test

Reply:
ok